src/mumble/Log.cpp

```diff
@@ -525,8 +525,8 @@ QString Log::imageToImg(const QByteArray &format, const QByteArray &image) {
 }
 
 QString Log::imageToImg(QImage img, int maxSize) {
-	if ((img.width() > 480) || (img.height() > 270)) {
-		img = img.scaled(480, 270, Qt::KeepAspectRatio, Qt::SmoothTransformation);
+	if ((img.width() > 800) || (img.height() > 600)) {
+		img = img.scaled(800, 600, Qt::KeepAspectRatio, Qt::SmoothTransformation);
```

Comment:
```suggestion
	constexpr int maxWidth  = 800;
	constexpr int maxHeight = 600;

	if ((img.width() > maxWidth) || (img.height() > maxHeight)) {
		img = img.scaled(maxWidth, maxHeight, Qt::KeepAspectRatio, Qt::SmoothTransformation);
```

Reply:
Looks good to me, I applied it through GitHub.

Do you want me to squash the commits as well or can you automatically do that whilst merging (GitLab can do that, not sure if GitHub can, too)?